cmd/bumpversion.go

```diff
@@ -43,7 +43,6 @@ func upgrade(targetOS string) (err error) {
 	} else {
 		engineCachePath = enginecache.ValidateOrUpdateEngine(targetOS, "")
 	}
-	engineCachePath = enginecache.ValidateOrUpdateEngine(targetOS, buildCachePath)
```

Comment:
Is the removal of this line related to version update/check?

Reply:
Nope, but I found it while going through the code. Just some random leftover I think.